packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -77,17 +79,44 @@ async function ensureLockedRecord(
   return attestationRecord
 }
 
+async function purgeExpiredRecords() {
+  const attestations = await kit.contracts.getAttestations()
+  const expiryTimeInSeconds = (await attestations.attestationExpiryBlocks()) * 5
+  const transaction = await sequelize!.transaction()
+  const AttestationTable = await getAttestationTable()
+  try {
+    await AttestationTable.destroy({
+      where: {
+        createdAt: {
+          [Op.lte]: moment()
+            .subtract(expiryTimeInSeconds, 'seconds')
+            .toDate(),
+        },
+      },
+      transaction,
+    })
+    await transaction.commit()
+  } catch (err) {
+    await transaction.rollback()
+  }
+}
+
 class AttestationRequestHandler {
   logger: Logger
+  identifier: string
   sequelizeLogger: (_msg: string, sequelizeLog: any) => void
   constructor(public readonly attestationRequest: AttestationRequest, logger: Logger) {
     this.logger = logger.child({ attestationRequest })
     this.sequelizeLogger = (msg: string, sequelizeLogArgs: any) =>
       this.logger.debug({ sequelizeLogArgs, component: 'sequelize' }, msg)
+    this.identifier = PhoneNumberUtils.getPhoneHash(
```

Comment:
I forget if the salt is optional, but if it is, I assume `getPhoneHash` works appropriately?

Reply:
Yes, salt is optional for this service and getPhoneHash works if its undefined